dashboard/app/views/levels/show.html.haml

```diff
@@ -22,6 +22,9 @@
   :javascript
     var appOptions = #{app_options.to_json};
   = render partial: 'levels/unplug', locals: {app: @game.app}
+- elsif @level.is_a?(Applab) && app_options['readonlyWorkspace'] && !app_options[:channel]
```

Comment:
it seems a bit weird to mix accessing the same hash via strings and symbols. Do you need to do this for some reason or is it an indifferent hash (where you can use either)?